spec/models/recipe_spec.rb

```diff
@@ -1,7 +1,10 @@
 require 'rails_helper'
+require 'validate_url/rspec_matcher'
```

Comment:
I think that you can move that to `rails_helper`, then you won't have to require it here.